README.md

```diff
@@ -0,0 +1,147 @@
+# Docker-snap
```

Comment:
I want to make a bunch of changes to the README, but will propose them after this initial version gets merged.